src/aap_eda/api/views/activation.py

```diff
@@ -296,6 +287,16 @@ def disable(self, request, pk):
                     deployment_type=settings.DEPLOYMENT_TYPE,
                 )
 
+            # update activation's details
+            activation.status = ActivationStatus.STOPPED
```

Comment:
The status should be STOPPING.
The whole block up to line 298 should be indent right to under the `if current_instance block` because the status should change only if the deactivate is applied.
We should have an else block to raise a response like 409.

Reply:
No, we should always update activation's status when it's disabled, no matter whether its activation instance needs to be cleaned up or not.

Reply:
Yes, the initial status should be `STOPPING`, `STOPPED` is marked only when disable is done